src/azure-cli/install.sh

```diff
@@ -127,7 +130,37 @@ install_using_apt() {
     fi
 }
 
-install_using_pip() {
+install_using_pip_strategy() {
+    local ver=""
+    if [ "${AZ_VERSION}" = "latest" ] || [ "${AZ_VERSION}" = "lts" ] || [ "${AZ_VERSION}" = "stable" ]; then
+        # Empty, meaning grab the "latest" in the apt repo
+        ver=""
+    else
+        ver="==${AZ_VERSION}"
+    fi
+
+    install_with_pipx "${ver}" || install_with_complete_python_installation "${ver}" || return 1
+}
+
+install_with_pipx() {
+    local ver="$1"
+
+    local REMOTE_USER_LOCAL_FOLDER="${_REMOTE_USER_HOME}/.local"
```

Comment:
FYI - [Bug: _REMOTE_USER_HOME is empty when executing a Feature where common-utils configures the user](https://github.com/devcontainers/cli/issues/331)

Reply:
I might want to put it somewhere global anyway 🤔 .  I just noticed that the existing pip branch here writes to a global location.